core/src/main/java/org/kiwix/kiwixmobile/core/page/bookmark/adapter/BookmarkViewHolder.kt

```diff
@@ -5,16 +5,18 @@ import kotlinx.android.synthetic.main.item_bookmark_history.favicon
 import kotlinx.android.synthetic.main.item_bookmark_history.title
 import org.kiwix.kiwixmobile.core.R
 import org.kiwix.kiwixmobile.core.base.adapter.BaseViewHolder
-import org.kiwix.kiwixmobile.core.page.bookmark.adapter.BookmarksAdapter.OnItemClickListener
 import org.kiwix.kiwixmobile.core.downloader.model.Base64String
 import org.kiwix.kiwixmobile.core.extensions.setBitmap
 import org.kiwix.kiwixmobile.core.extensions.setImageDrawableCompat
+import org.kiwix.kiwixmobile.core.page.adapter.OnItemClickListener
+import org.kiwix.kiwixmobile.core.page.adapter.PageRelated
 
 class BookmarkViewHolder(
   override val containerView: View,
   private val itemClickListener: OnItemClickListener
-) : BaseViewHolder<BookmarkItem>(containerView) {
-  override fun bind(item: BookmarkItem) {
+) : BaseViewHolder<PageRelated>(containerView) {
```

Comment:
The ViewHolders both do the exact same thing I think, there could probably be only 1